GameData/RealSolarSystem/RSSKopernicus.cfg

```diff
@@ -896,9 +896,9 @@
 		}
 		PQS
 		{
-			deactivateAltitude = 130000
-			fadeStart = 80000
-			fadeEnd = 120000
+			deactivateAltitude = 101000

```

Comment:
Also, the PQS needs to have higher fade values than scaled space. That way scaled space fades in before PQS fades out, and vice versa. Otherwise you can get like 50% opacity planets.